src/web/components/Branding.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { LinkStyle } from '@root/src/amp/components/elements/TextBlockComponent';
+import { textSans } from '@guardian/src-foundations/typography';
+
+import { css } from 'emotion';
+import { palette } from '@guardian/src-foundations';
+
+const brandingStyle = (pillar: Pillar) => css`
+    padding-bottom: 10px;
+    ${LinkStyle(pillar)}
+
+    a, a:hover {
+        display: block;
+        border-bottom: none;
+        ${textSans.xsmall()}
+    }
```

Comment:
This will probably go away if we use `Link` but in general, with css-in-js it's a good rule to never use css selectors. They create coupling which can slowly grow in bad ways...